neo/Consensus/ConsensusService.cs

```diff
@@ -181,8 +181,26 @@ private void LocalNode_InventoryReceived(object sender, IInventory inventory)
                 lock (context)
                 {
                     if (payload.ValidatorIndex == context.MyIndex) return;
-                    if (payload.Version != ConsensusContext.Version || payload.PrevHash != context.PrevHash || payload.BlockIndex != context.BlockIndex)
+
+                    if (payload.Version != ConsensusContext.Version)
                         return;
+                    if (payload.PrevHash != context.PrevHash || payload.BlockIndex != context.BlockIndex)
+                    {
+                        // Request blocks
+
+                        if (Blockchain.Default?.Height < payload.BlockIndex)
```

Comment:
I think it should be `if (Blockchain.Default?.Height + 1 < payload.BlockIndex)` because `(Blockchain.Default?.Height < payload.BlockIndex)` is always `true`.

Reply:
@erikzhang The right command should be `getblocks` or `getheaders` ?

Reply:
`getblocks`